cdc/sink/codec/canal.go

```diff
@@ -538,19 +537,17 @@ func NewCanalEventBatchEncoder() EventBatchEncoder {
 }
 
 type canalEventBatchEncoderBuilder struct {
-	opts map[string]string
+	config *Config
```

Comment:
Why do we store this parameter if we don't need it? It doesn't look like we are using this parameter.